### DISCUSSION TITLE
Path Issue with gmsh While Trying to Install neper in Conda Environment

### DISCUSSION THREAD

**Discussion by ParadoxPrism117** [neperfepx/neper#938] [Q&A] [answered]
Hi,

I am running into an issue trying to install neper on Ubuntu 22. I've installed and checked that gmsh was up to date but every time I try to run the mesh module I get "Error: Option '-gmsh' does not provide a valid path to the gmsh binary." the tessellation and visualization modules both seem to work without issue. Also, when I run ctest, pretty much every test fails. Could this stem from me trying to do the installation in a conda virtual environment? Or is there some way that I can fix this path issue?

**Comment by rquey** [comment, marked as answer]
There is a complete tutorial for installation on Ubuntu 22: https://neper.info/doc/tutorials/install_ubuntu22.html.

About conda, I don't know...
